extensions/amp-iframe/0.1/validator-amp-iframe.protoascii

```diff
@@ -72,6 +72,8 @@ tags: {  # <amp-iframe>
     name: "srcdoc"
     mandatory_oneof: "['src', 'srcdoc']"
   }
+  # <amp-bind>
+  attrs: { name: "[src]" }
```

Comment:
Good point. This would result in an invalid doc after amp-bind evaluates `[src]="someExpression"` which adds the `src` attribute to the element. To address this, I added a line to amp-iframe.js that removes the `srcdoc` attribute when applying `[src]`.

I think this aligns with the intent of the mutual exclusivity of `src` and `srcdoc`, which it appears is to not allow users to specify an extraneous iframe data source.